src/BatteryStateOfChargeService.h

```diff
@@ -45,5 +45,12 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
 
+    double AmpHoursUsed_;
+    double inCurrent_;
```

Comment:
`inCurrent_` is only used in one function and doesn't really need to stored between function calls, it would be better suited as a variable in `addData` rather than a private member.